ote/src/cljs/ote/views/service_viewer.cljs

```diff
@@ -493,32 +531,362 @@
+   [spacer]])
+
+(defn- accessibility-info [title accessibility accessibility-description accessibility-info-url information-service-accessibility]
+  [:section
+   [info-sections-1-col title
+    [:div
+     [common-ui/information-row-with-option
+      (tr [:service-viewer :accessibility])
+      (string/join ", " (map (fn [key] (tr [:enums ::t-service/accessibility key])) accessibility))
+      true]
+     [common-ui/information-row-with-option (tr [:field-labels :terminal :ote.db.transport-service/information-service-accessibility])
+      (string/join ", " (map (fn [key] (tr [:enums ::t-service/information-service-accessibility key])) information-service-accessibility))
+      true]
+     [common-ui/information-row-with-option (tr [:service-viewer :accessibility-website]) accessibility-info-url true]
+     [information-row-with-selection (tr [:field-labels :transport-service-common ::t-service/accessibility-description]) (format-descriptions accessibility-description) true]]]
+   [spacer]])
+
+(defn- parking-restrictions [title data]
+  (let [parking-restricted? (or (not (empty? data)) false)
+        restriction-time-type (cond
+                                (and
+                                  (not (nil? (:minutes data)))
+                                  (> (:minutes data) 0)) :minutes
+                                (and
+                                  (not (nil? (:hours data)))
+                                  (> (:hours data) 0)) :hours
+                                (and
+                                  (not (nil? (:days data)))
+                                  (> (:days data) 0)) :days
+                                :default :hours)
+        restriction-value (restriction-time-type data)]
+    [:section
+     [info-sections-1-col title
+      [:div
+       [common-ui/information-row-with-option
+        (tr [:service-viewer :parking-restricted-text])
+        (tr [:service-viewer :parking-restricted? parking-restricted?]) false]
+      [common-ui/information-row-with-option
+       (tr [:field-labels :parking :ote.db.transport-service/maximum-stay])
+       (str restriction-value " " (string/lower-case (tr [:common-texts :time-units restriction-time-type]))) false]]]
+     [spacer]]))
+
 (defn service-view
   [e! {{to :transport-operator ts :transport-service} :service-view}]
-  (let [interfaces (::t-service/external-interfaces ts)
-        warnings (get-in ts [::t-service/passenger-transportation ::t-service/luggage-restrictions])
-        real-time-info-data (get-in ts [::t-service/passenger-transportation ::t-service/real-time-information])
-        pre-booking-data (get-in ts [::t-service/passenger-transportation ::t-service/advance-reservation])
-        booking-data (get-in ts [::t-service/passenger-transportation ::t-service/booking-service])
+  (let [service-sub-type (get ts ::t-service/sub-type)
```

Comment:
`let` could be put inside the `if`so that all the bindings are skipped when they're not needed.